### PR TITLE
Admin orders

### PR DESCRIPTION
**Pull request by sabondano**
- closes # 58

- User story: https://waffle.io/michael-reeves/redrum_nursery/cards/55c8f0e3e0e0b13200af45dd

- This change allows an admin to change an order's status.

### CODE REVIEW

File: spec/features/admin_sees_all_orders_spec.rb [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+
+    within("tr", text: "# #{@order_4.id}") do
+      # expect(page).to have_xpath("//a[@href=\"#{admin_order_path(@order_1)}\"]")
+      expect(page).to have_content("Completed")
+      expect(page).to have_content("$9.99")
+      expect(page).to have_content("July  5, 2015 at  9:33 PM")
+    end
+
+    status = find("table").first(".order-status").text
+      expect(status).not_to eq("Ordered")
```
Comment (imwithsam) [inline spec/features/admin_sees_all_orders_spec.rb]: Commented out code left for upcoming merge.

File: spec/features/admin_sees_all_orders_spec.rb [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+      expect(page).to have_link("Completed")
+    end
+
+    within("tr", text: "# #{@order_3.id}") do
+      expect(page).to_not have_link("Ordered")
+      expect(page).to_not have_link("Paid")
+      expect(page).to_not have_link("Completed")
+    end
+
+     within("tr", text: "# #{@order_4.id}") do
```
Comment (imwithsam) [inline spec/features/admin_sees_all_orders_spec.rb]: Looks like Line 222 may be off one space.

File: spec/features/admin_sees_all_orders_spec.rb [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+
+
+# As an Admin
+# When I visit the dashboard
+# Then I can see a link to a listing of all orders
+# And I can see the total number of orders for each status ("Ordered", "Paid", "Cancelled", "Completed")
+# And I can see a link for each individual order
+# And I can filter orders to display by each status type  ("Ordered", "Paid", "Cancelled", "Completed")
+# And I have links to transition the status
+#
```
Comment (imwithsam) [inline spec/features/admin_sees_all_orders_spec.rb]: Do we need to keep these comments in here?

**Comment by imwithsam** [comment]
Looks great! Just made a few minor comments on formatting and possible leftover comments.

**Comment by mdorrance** [comment]
Looks good to me.

**Pull request merged**